src/sys/socket/consts.rs

```diff
@@ -93,11 +277,16 @@ mod os {
         pub flags MsgFlags: libc::c_int {
             MSG_OOB,
             MSG_PEEK,
+            MSG_DONTWAIT,
+            #[cfg(not(target_os = "dragonfly"))]
             MSG_CTRUNC,
```

Comment:
`MSG_CTRUNC`, `MSG_EOR`, and `MSG_TRUNC` are all available in `libc` for DragonflyBSD.